spyder/utils/palette.py

```diff
@@ -39,8 +39,8 @@ class SpyderPaletteDark:
 
     # Icon colors
     ICON_1 = Gray.B140
-    ICON_2 = Blue.B70
-    ICON_3 = Green.B60
+    ICON_2 = Blue.B80
+    ICON_3 = Green.B80
```

Comment:
Is this right? I mean `ICON_2` and `ICON_3` point to the same color now.

Reply:
Thanks for checking! I thought I made a typo, but it's correct because they are the same value of different colors. One is Blue and the other is Green.

Reply:
My bad! I read these lines too quickly and thought they were poitining to the same color.